docs/docsite/rst/getting_started_ee/run_community_ee_image.rst

```diff
@@ -0,0 +1,26 @@
+.. _running_community_ee_image:
+
+Running a playbook using the community EE image
+===============================================
+
+There is the ``community-ee`` image built with ``ansible-core`` and a set of popular collections
+which you can use to run your playbooks without building your custom EE.
+
+Run the following command to see the included collections.
+
+.. code-block:: bash
+
+  ansible-navigator collections --execution-environment-image ghcr.io/ansible/community-ee:latest
+
+To try out the EE, run the following ad hoc command against localhost inside the container.
+
+.. code-block:: bash
+
+  ansible-navigator exec "ansible localhost -m setup" --execution-environment-image ghcr.io/ansible/community-ee:latest --mode stdout
```

Comment:
The section says "Running a playbook", but now you removed the playbook :-)

Reply:
hehe, well spotted! Replaced `playbook` with more abstract `ansible`, PTAL. thanks!

Reply:
I would also show how to use the EE with a playbook and an inventory file (probably in a new section). Running direct commands on localhost is nice, but not something that generalizes well... if someone simpy replaces `ansible localhost -m setup` with `ansible-playbook -i inventory playbook.yml`, or even start to build a Docker image upon the community-ee image with their playbook and inventory in it, they won't use the EE as intended.

Having a direct command as the first step to use an EE is fine, but how to properly use it should be shown right after that (and not only after clicking a link or having to search for more information on it).

Reply:
Ah, great feedback, makes a lot of sense. thanks!
- Added an example (not to overwhelm the doc, it's only about localhost but i put a ref to the example from another page if users need it right now)
- Also added the community-ee as a base image to the building guide as by default it would use navigator's default image already containing a lot of other stuff and not aiming to be a base image i think

PTAL